arraykit.pyi

```diff
@@ -25,5 +25,6 @@ def shape_filter(__array: np.array) -> np.ndarray: ...
 def column_2d_filter(__array: np.array) -> np.ndarray: ...
 def column_1d_filter(__array: np.array) -> np.ndarray: ...
 def row_1d_filter(__array: np.array) -> np.ndarray: ...
+def array_deepcopy(__array: np.array, memo: tp.Dict[int, tp.Any]) -> np.ndarray: ...
```

Comment:
```suggestion
def array_deepcopy(__array: np.array, memo: tp.Dict[int, np.ndarray]) -> np.ndarray: ...
```

Reply:
I am not sure about this. I think in a nested `deepcopy` call, the `memo` dict will pick up references to any mutable that needs deep copying and store it in this single dict. So in many contexts (i.e., `deepcopy([np.array([3,4])])`) the memo dict will have things other than arrays.